### ISSUE TITLE
Unexpanded autoconf macro @LIB_RT@ in 3.0.2

### ISSUE THREAD

**Issue by jasontibbitts** [cyrusimap/cyrus-imapd#2039]
This is new in 3.0.2 and appears to be a side effect of ef5d65d739084da61d90eb05ed4e41e7e7faf8f6.

Five files in the tarball files contain the string "@LIB_RT@": Makefile.PL.in and Makefile.PL for the managesieve and imap Perl modules, as well as libcyrus.pc.in.  No LIB_RT substitution is defined in the autoconf system, so these strings are left in when those *.in files are processed.
```
LIBS'      => ["$LIB_SASL -lssl -lcrypto -lssl -lcrypto  @LIB_RT@ -luuid -lz"],
```
(Also, it appears that all of the Makefile.PL files are regenerated as part of the build process so I'm not sure if they need to be included in the tarball.)

It doesn't appear that this breaks either of the Perl modules, though; that string doesn't make it into the resulting Makefiles when perl generates them.  It does, however, make it into the final libcyrus.pc file, and that would appear to be a bug.

**Comment by elliefm** [comment]
Oh damn, I thought I got all of those :/

**Comment by elliefm** [comment]
And I thought I'd pulled the Makefile.PL files out of the tarball as well, I wonder if I forgot to merge a branch :(

**Comment by elliefm** [comment]
The Makefile.PL thing is fixed now on 3.0 and master branches.

**Comment by elliefm** [comment]
I've reverted the Makefile.PL change for now because it confuses the perl modules' build systems, causing our top level `make distcheck` to fail unnecessarily.

The @LIB_RT@ references should be all cleaned up now.
